dockstore-webservice/src/main/java/io/dockstore/webservice/helpers/FileFormatHelper.java

```diff
@@ -66,4 +75,18 @@ public static void updateFileFormats(Set<? extends Version> versions, final File
         });
         return fileFormatsFromDB;
     }
+
+    public static String calcSHA1(String content) {
+        //return org.apache.commons.codec.digest.DigestUtils.sha1Hex(content);
+        String sha1 = null;
+        try {
+            MessageDigest messageDigest = MessageDigest.getInstance("SHA-1");
+            messageDigest.update(content.getBytes("UTF-8"), 0, content.length());
+            sha1 = DatatypeConverter.printHexBinary(messageDigest.digest());
+            sha1.toLowerCase();
```

Comment:
1. This isn't doing anything -- toLowerCase returns a new string which you aren't using. It doesn't modify the existing one.
2. Why are you doing this? Not saying you shouldn't, but can you add a comment explaining why?

Reply:
had accidentally put it in the wrong place initially. But I did have it return with toLowerCase just because the checksums we receive via quay, dockerhub for images are lowerecased and I thought it'd be better to match that.